package.json

```diff
@@ -54,5 +54,8 @@
     "shell-utils": "1.x.x",
     "unified": "^10.1.0"
   },
-  "version": "20.1.2"
+  "version": "20.1.2",
+  "dependencies": {
+    "docusaurus-plugin-sass": "^0.2.3"
```

Comment:
Why do we need it here at all in the root project?

Reply:
Thanks for the catch there. Removed and it's now in the website root package.json.